index.html

```diff
@@ -81,6 +81,42 @@ <h2 id="subtitle">
         This is a work in progress.
       </p>
     </section>
+    <section class="informative">
+      <h2 id="intro">
+        Introduction
+      </h2>
+      <p>
+        Specifications can define features that are explicitly identified as a [=powerful
+        feature=]. These features are said to be "powerful" in that they can have significant
+        privacy, security, and performance implications. As such, users rely on user agents to deny
+        sites the ability to use these features until they have given express permission, and
+        usually only granting this ability for a limited amount of time. Express permission to
+        allow a site to use a powerful feature is generally given and controlled through browser
+        UI, as illustrated below.
+      </p>
+      <figure>
+        <img src="docs/assets/images/sample-prompts.png" alt=
+        "On the left, a mockup of a notifications prompt with an allow and don't allow buttons that states 'the website example.com would like to send you notifications'. On the right, a prompt near the URL bar asking to give permission to the camera and microphone to the example site.">
+        <figcaption>
+          Sketches of possible permission prompt types
+        </figcaption>
+      </figure>
+      <p>
+        In this sense, a permission represents the current state of user consent for certain types
+        of features, and particularly "powerful features". Ultimately users retain control of these
+        permissions (although the user agent may make decisions to reduce the impact of having
+        granted a permission, e.g. by reducing Notification abuse, or automatically expiring
+        granted permissions if a user doesn't visit a website for some time), and have the ability
+        to limit or revoke permissions through user preferences.
```

Comment:
```suggestion
        In this sense, a permission represents the current state of user consent for certain types
        of features, and particularly "powerful features". Ultimately the user retains control of these
        permissions and have the ability to manually grant or deny permissions through user preferences. 
        Further, user agents assist users in managing permissions by, for example, hiding and automatically
        denying certain permission prompts that would otherwise be a nuisance, and automatically expiring
        granted permissions if a user doesn't visit a website for some time.
```

Reply:
I want to avoid calling out Notifications specifically.

Reply:
i like it, thanks